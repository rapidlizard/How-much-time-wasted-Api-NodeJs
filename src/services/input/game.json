{
    "appid": 1,
    "playtime_forever": 1000,
    "playtime_windows_forever": 0,
    "playtime_mac_forever": 0,
    "playtime_linux_forever": 0
}