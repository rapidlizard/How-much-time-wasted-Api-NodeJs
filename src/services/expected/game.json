{
    "appid": 1,
    "playtime": 1000
}